app/models/comparison/period.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: comparison_periods
+#
+#  created_at :datetime         not null
+#  end_date   :date             not null
+#  id         :bigint(8)        not null, primary key
+#  label      :string           not null
+#  start_date :date             not null
+#  updated_at :datetime         not null
+#
+class Comparison::Period < ApplicationRecord
```

Comment:
I'd originally thought this would define 2 date ranges for current and previous period along with labels for both.